src/schemas/json/github-action.json

```diff
@@ -119,6 +120,15 @@
           "items": {
             "type": "string"
           }
+        },
+        "post": {
```

Comment:
this is not documented but it's used in an official action: https://github.com/actions/cache/blob/37c45447e4220b80ffd0f1b418c35f1a1808d204/action.yml#L20-L21